code/datums/manufacturing.dm

```diff
@@ -1352,159 +1352,176 @@ ABSTRACT_TYPE(/datum/manufacture)
 /datum/manufacture/robo_frame
 	name = "Cyborg Frame"
 	item_paths = list("MET-2")
-	item_amounts = list(18)
+	item_amounts = list(ROBOT_FRAME_COST * 10)
```

Comment:
Why are these all `* 10`? Why not just have the defines be 10x larger?

Reply:
Manufacturing for some reason has the material values as 10 times larger as the material value used in the item. 

For exapmle in order for sheet to cost 0.1 metal it has to have 1 in the recipe and in order to be reclaimed for the same amount it has to have a material amount of 0.1